apps/wallet/src/background/accounts/index.ts

```diff
@@ -336,5 +337,32 @@ export async function accountsHandleUIMessage(msg: Message, uiConnection: UiConn
 		);
 		return true;
 	}
+	if (isMethodPayload(payload, 'removeAccount')) {
+		const { accountID } = payload.args;
+		const db = await getDB();
+		await db.transaction('rw', db.accounts, db.accountSources, async () => {
+			const account = await db.accounts.get(accountID);
+			if (!account) {
+				throw new Error(`Account with id ${accountID} not found.`);
+			}
+			const accountSourceID =
+				'sourceID' in account && typeof account.sourceID === 'string' && account.sourceID;
+			await db.accounts.delete(account.id);
+			if (accountSourceID) {
+				const totalSameSourceAccounts = await db.accounts
+					.where('sourceID')
+					.equals(accountSourceID)
+					.count();
```

Comment:
another small callout - we might want to re-think ordering by created date for mnemonic derived accounts at some point: 


https://github.com/MystenLabs/sui/assets/122397493/ccf6b453-fda8-410a-b814-e3806b82376d

Reply:
Yeah it felt weird initially but should be ok (it feels like this because we know the derivation path). We can sort with the derivation path but I think it's ok as is it is (at least for now)